SUSE/rmt
# Fix: Set correct user for service and cli

## Description

RMT server service uses non-root user which causes access permissions issues. This PR fixes this issue.

Fixes bsc#1215176

## Change Type

*Please select the correct option.*


## Checklist

*Please check off each item if the requirement is met.*


## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.


How to test
-----------------

 **Pre-requisite**: Setup a vm with nginx and ssl for client script setup. Also, please mirror the client system  product on your test rmt.

1. Pull the `rmt-client-setup`  on your client system: 

`curl <ip>/tools/rmt/client/setup`

2. Register your client: 

`./rmt-client-setup https://<ip>`

This should work without any issue,
